packages/design-system-react-native/src/components/BadgeStatus/BadgeStatus.types.ts

```diff
@@ -0,0 +1,57 @@
+import { ViewProps } from 'react-native';
+
+/**
+ * The status of BadgeStatus
+ */
+export enum BadgeStatusStatus {
+  Active = 'active',
+  PartiallyActive = 'partiallyactive',
+  Inactive = 'inactive',
+  New = 'new',
+  Attention = 'attention',
+}
```

Comment:
nit: we could add comments here to explain what these mean but it's a pretty niche component